src/Utils.cpp

```diff
@@ -74,6 +74,15 @@ float Utils::cached_t_quantile(int v) {
     return z_lookup[z_entries - 1];
 }
 
+float Utils::erfinv_approx(float x)
+{
+    float sign = x > 0 ? 1 : -1;
```

Comment:
The coding style guidelines say to use `auto` if possible so probably need to use `0.0f` etc.
Same for rest of function.